src/map/npc.cpp

```diff
@@ -1123,12 +1123,6 @@ TIMER_FUNC(npc_secure_timeout_timer){
 
 	if( DIFF_TICK(cur_tick,sd->npc_idle_tick) > (timeout*1000) ) {
 		pc_close_npc(sd,1);
-	} else if(sd->st && (sd->st->state == END || sd->st->state == CLOSE)){
```

Comment:
Removing this part looks wrong.

Reply:
Any other roles in this part? Because in KRo secure timer doesn't being stopped on ```close``` or ```end```